udata/tests/frontend/test_swagger.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import json
+
+from . import FrontTestCase
+
+
+class SwaggerBlueprintTest(FrontTestCase):
+
+    def test_swagger(self):
```

Comment:
Be more specific: you're not testing the entire `swagger.json`, only the resource type